charts/rancher-cis-benchmark/0.2.1/questions.yaml

```diff
@@ -0,0 +1,2 @@
+rancher_min_version: 2.6.0
+rancher_max_version: 2.6.0
```

Comment:
This is what the 0.2.0 questions.yaml should be. The 0.2.1 should be just the min.

Reply:
And I think the min for 0.2.1 should be rancher 2.6.1, correct?

Reply:
Technically yes, but it would be ok if you want to make it available in 2.6.0.